ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/poe.j2

```diff
@@ -0,0 +1,65 @@
+{# doc - poe #}
+{% set POE_CLASS_MAP = {0: "15.40", 1: "4.00", 2: "7.00", 3: "15.40", 4: "30.00", 5: "45.00", 6: "60.00", 7: "75.00", 8: "90.00"} %}
+{% set poe_global_defaults = namespace(reboot_action="power-off", interface_shutdown_action="maintain", lldp_negotiate=true) %}
+{% set ethernet_interfaces_poe = [] %}
+{% for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
+{%     if ethernet_interface.poe is arista.avd.defined %}
+{%         do ethernet_interfaces_poe.append(ethernet_interface) %}
+{%     endif %}
+{% endfor %}
+{% if (poe is arista.avd.defined) or (ethernet_interfaces_poe | length > 0) %}
+## Power Over Ethernet (PoE)
+
+### PoE Summary
+{%     if poe is arista.avd.defined %}
+
+#### PoE Global
+
+| Reboot Action | Shutdown Action | LLDP Negotiation |
+| ------------------- | -------------------- | ----------------------|
+{# Get global poe settings for defaults #}
+{%         set poe_global_defaults.reboot_action = poe.reboot.action | arista.avd.default(poe_global_defaults.reboot_action) %}
+{%         set poe_global_defaults.interface_shutdown_action = poe.interface_shutdown.action | arista.avd.default(poe_global_defaults.interface_shutdown_action) %}
+{#         get lldp transmit power via mdi #}
+{%         if lldp.tlvs is arista.avd.defined %}
+{%             for tlv in lldp.tlvs if tlv.name == 'power-via-mdi' %}
+{%                 set poe_global_defaults.lldp_negotiate = tlv.transmit %}
+{%             endfor %}
+{%         endif %}
+| {{ poe_global_defaults.reboot_action }} | {{ poe_global_defaults.interface_shutdown_action }} | {{ poe_global_defaults.lldp_negotiate }} |
+{%     endif %}
+{%     if ethernet_interfaces_poe | length > 0 %}
+
+#### PoE Interfaces
+
+{%         if ethernet_interfaces_poe | length > 0 %}
+| Interface | PoE Enabled | Priority | Limit | Reboot Action | Link Down Action | Shutdown Action | LLDP Negotiation | Legacy Detection |
+| --------- | --------- | --------- | ----------- | ----------- | ----------- | ----------- | --------- | --------- |
+{%             for ethernet_interface in ethernet_interfaces_poe | arista.avd.natural_sort('name') %}
+{%                 set poe = ethernet_interface.poe %}
+{%                 set enabled = not poe.disabled | arista.avd.default(true) %}
```

Comment:
Hmm is it working as expected ?  I think this will give a default of false as it is understood as

```
{%                 set enabled = not (poe.disabled | arista.avd.default(true)) %}
```

But you would rather want (not sure if it works)

```
{%                 set enabled = (not poe.disabled) | arista.avd.default(true) %}
```

as by default poe is enabled right?

cf order of precedence for operators in Jinja

Reply:
It actually works as is but I think using `{%                 set enabled = (not poe.disabled) | arista.avd.default(true) %}` is probably more obvious so I'll update accordingly.

Reply:
Looks like `set enabled = not (poe.disabled | arista.avd.default(false))` gets the job done

Reply:
I agree that this does the job eventhough it is quite confusing  to read (but it is Jinja's fault :) )